src/components/throwtimer/css.module.scss

```diff
@@ -0,0 +1,27 @@
+.rightAlign{
+    position: relative;
+    left: 175px;
+    width: 100%;
+  }
+
+  .countdownTimer{
+    font-size: 50px;
+    //color: red;
+  }
+
+  .circular{
```

Comment:
Any css class you find yourself reusing more than once suggest that they need to be placed in a global file to be access by all components instead of being recreated more than once.

Reply:
Ahh that makes sense ty